src/Nancy/HttpLinkBuilder.cs

```diff
@@ -0,0 +1,49 @@
+namespace Nancy
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Builds an RFC 5899 Link HTTP header as a <see cref="List{T}" /> of <see cref="HttpLink" /> objects.
+    /// </summary>
+    /// <seealso cref="HttpLink" />
+    public class HttpLinkBuilder : List<HttpLink>
```

Comment:
No, but `List<T>` has a nice `.Add()` method that makes a lot of sense to a `Builder` class, don't you agree?